certbot/main.py

```diff
@@ -809,16 +809,11 @@ def install(config, plugins):
         domains, _ = _find_domains_or_certname(config, installer)
         le_client = _init_le_client(config, authenticator=None, installer=installer)
         _install_cert(config, le_client, domains)
+        return None
     else:
         raise errors.ConfigurationError("Path to certificate or key was not defined. "
             "If your certificate is managed by Certbot, please use --cert-name "
             "to define which certificate you would like to install.")
-    return None
-
-    if enhancements.are_requested(config):
```

Comment:
Why are we deleting this block?

Reply:
Dead code.

Reply:
I think it's only dead code because of the addition of `return None` at https://github.com/certbot/certbot/pull/6905/files#diff-0b4a662e2c7b44bd47156f3d976ed698R812.

I think this code should be added back and `return None` added to the end of this function after this `if` block.

Reply:
Indeed. I added again the code block, with premature return